src/App.tsx

```diff
@@ -1,61 +1,120 @@
+/* eslint-disable no-console */
 import './App.scss';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoWithUser, TodoWithoutUser } from './types/Todo';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserById(todo: TodoWithoutUser) {
+  return usersFromServer.find(({ id }) => id === todo.userId) || null;
+}
```

Comment:
```suggestion
function getUserById(userId: number): User {
  return usersFromServer.find(({ id }) => id === userId) || null;
}
```
Function is called `getUserById`, but it receives a todo object, that's misleading, it should receive just a `userId`.